components/homme/src/share/namelist_mod.F90

```diff
@@ -209,7 +211,12 @@ subroutine readnl(par)
 #ifdef CAM
       se_partmethod,     &
       se_topology,       &
+      se_geometry,       &
       se_ne,             &
+      se_ne_x,           &
+      se_ne_y,           &
+      se_lx,             &
+      se_ly,             &
       se_limiter_option, &
 #else
       qsize,             &         ! number of SE tracers
```

Comment:
All these modifications look fine to me! Do we want to add the ability to set Lx/Ly in standalone planar HOMME as well? Since standalone HOMME is only (to my knowledge) ever run with test cases as initial conditions (which have a given Lx/Ly) this is probably not needed...

Reply:
I'll leave this up to you.  I never run HOMME in standalone mode so I think you're better equipped to answer this; but if what you said is in fact true then yes, it is probably not needed.

Reply:
Let's not worry about it for now- it is easy enough to add the capability if needed.